Build/AssemblyVersion.cs

```diff
@@ -10,22 +10,19 @@
 // Properties\AssemblyInfo.cs file and remove any lines duplicating the ones below.
 // (See also AssemblyInfoCommon.cs in this same directory.)
 
-[assembly: AssemblyVersion(AssemblyVersionInfo.StableVersion)]
-[assembly: AssemblyFileVersion(AssemblyVersionInfo.Version)]
+[assembly: AssemblyVersion(AssemblyVersionInfo.Version)]
 
 class AssemblyVersionInfo {
+    
     // This version string (and the comments for StableVersion and Version)
     // should be updated manually between major releases.
     // Servicing branches should retain the value
-    public const string ReleaseVersion = "2.0";
+    public const string MajorVersion = "1";

```

Comment:
We put major back to 1. Do we need to worry about the B2B from the old version(2.x) ?